### ISSUE TITLE
Does <meta charset="..."> handled by the server?

### ISSUE THREAD

**Issue by tucsu** [kzahel/web-server-chrome#123]

**Comment by kzahel** [comment]
No. Is that what web servers should do? Read and parse HTML/XML and look for the character set? I guess that makes sense. I think I am just always setting utf-8 by default at the moment, but doing something more clever like that would make sense. Do you have a link to source code in another web server such as apache/nginx where they detect character set of html file?

**Comment by kzahel** [comment]
The original Chrome App is no longer supported. The app has been rebuilt from scratch — see https://ok200.app for the new versions and to sign up for updates.
